sdk/cognitivelanguage/Azure.AI.Language.sln

```diff
@@ -4,6 +4,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 17.0.31815.197
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.Language.QuestionAnswering", "Azure.AI.Language.QuestionAnswering\src\Azure.AI.Language.QuestionAnswering.csproj", "{9507A18C-F21A-4305-80B8-8E9DB4E5D351}"
+	ProjectSection(ProjectDependencies) = postProject
+		{8DAB9728-0F12-40DF-A5B7-EFD602AF3C0E} = {8DAB9728-0F12-40DF-A5B7-EFD602AF3C0E}
+	EndProjectSection
```

Comment:
What does this do? If not necessary, I'd back this change out.

Reply:
I believe this adds a project dependency for QA on the Azure.Core project. It was automatically added when adding the reference the project reference. Should I still remove it?

Reply:
No, assuming we keep the ProjectReference, but I'm surprised a new project wasn't added to the SLN itself. That's why this diff block confused me.